sdk/keyvault/keyvault-keys/CHANGELOG.md

```diff
@@ -1,8 +1,13 @@
 # Release History
 
-## 4.0.1 (TBD)
-
-- Updated to use OpenTelemetry 0.2 via `@azure/core-tracing`
+## 4.0.1 (2019-12-02)
+
+- Updated dependencies to their latest available versions.
+- Fixed the support of dotenv while testing.
+- Cleaned up the docs from unsupported syntax.
```

Comment:
I don't know what this means

Reply:
What I meant here is that we were using syntax unsupported for tsdocs. Is it better if I remove this line? (and the other identical one)

Reply:
I'd just remove it. It doesn't seem like something people would care about